malibu-advanced.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Malibu Advanced
+nav_order: 6
+permalink: /malibu-advanced
+---
+# {{page.title}}
+
+## Introduction
+
+While Malibu provides a basic platform to build websites quickly, Malibu Advanced is an upgraded version with performance improvements and some of the third-party integrations commonly used by publishers. It is also integrated with our internal design library called Arrow which gives the look and feel of a fully functional website.
+
+Malibu Advanced has the following add-ons over Malibu:
+
+- **DFP Ads:** We’ve tried multiple approaches to integrate the DFP ads, each of them having its pros and cons. After numerous experiments, we landed on using vanilla JavaScript which provided us with the easiest implementation and customisation along with reasonably less impact on the Google Lighthouse scores. For detailed insights on the DFP integration, please check [here](https://developers.quintype.com/malibu/tutorial/dfp-ads.html).
+
+- **OneSignal Integration:** While Bold supports multiple push notification services, we have integrated OneSignal on Malibu Advanced as an example. You can see the documentation [here](https://developers.quintype.com/malibu/tutorial/onesignal.html). If you want to use any other service, this can be used as a reference for implementation.
+
+- **GA and GTM:** Malibu Advanced ships with GA and GTM out of the box. All you have to do to initialise it in your app is to add the corresponding IDs in the configuration file in black knight.
+
+- **Arrow design library:** Please refer to the docs [here](https://developers.quintype.com/malibu/tutorial/arrow-integration-malibu-advanced.html) on the Arrow integration.
+For detailed insights on what Arrow is, please check the [Arrow docs](https://developers.quintype.com/quintype-node-arrow/?path=/story/introduction--getting-started).
+
```

Comment:
can we mention the login feature ? also, we have a basic design for the story page, author, search etc.

Reply:
@Athira001 
1. for login: I will be mentioning that we have integrated, will give the brk-apis link that we have and also mention that we will be updating the implementation docs soon
2.I think the design need not be mentioned, since in my opinion, the design is basically done so that the showcasing can be done well, am pretty sure, no one's gonna use the design as is.